infrastructure/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM openjdk:8-jdk-alpine3.8
+ARG BASE_IMAGE=openjdk:8-jdk-alpine3.8
```

Comment:
will command line override for BASE_IMAGE work if the variable is set here explicity?